ArcadiaTeamsBot/Dialogs/NewRequestDialog.cs

```diff
@@ -1,63 +1,288 @@
 ﻿namespace ArcadiaTeamsBot.Dialogs
 {
+    using System;
     using System.Collections.Generic;
+    using System.Linq;
     using System.Threading;
     using System.Threading.Tasks;
 
+    using AdaptiveCards;
+
+    using ArcadiaTeamsBot.CQRS.Abstractions;
+    using ArcadiaTeamsBot.CQRS.Abstractions.Commands;
+    using ArcadiaTeamsBot.ServiceDesk.Abstractions;
+    using ArcadiaTeamsBot.ServiceDesk.Abstractions.DTOs;
+    using ArcadiaTeamsBot.ServiceDesk.Requests.RequestType;
+    using ArcadiaTeamsBot.ServiceDesk.Requests.RequestTypeFactory;
+
+    using MediatR;
+
     using Microsoft.Bot.Builder;
     using Microsoft.Bot.Builder.Dialogs;
     using Microsoft.Bot.Schema;
 
-    public class NewRequestDialog : ComponentDialog
+    using Newtonsoft.Json.Linq;
+
+    internal class NewRequestDialog : ComponentDialog
     {
-        private const string Back = "Back";
+        private const string Submit = "Submit";
+        private const string Cancel = "Cancel";
+        private const string Yes = "Yes";
+        private const string No = "No";
+        private const string Button = "Button";
```

Comment:
I suppose it should be the same as property name in line 200. In that case you'd better introduce new private interface like `interface IButtonData { string Button { get; set; } }` and use `nameof(IButtonData.Button)`. Then you will never forgot to change value in one place after changing in another.